### PR TITLE
App logic

### PR DESCRIPTION
**Pull request by elberthcorniell**
In this pull request we want to add the next methods:

- Operate.
- Calculate

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = ({ total = '', next = '', operation = '' }) => {
+  const x = new Big(total);
+  const y = new Big(next);
+  let result;
+  switch (operation) {
+    case '%':
```
Comment (Torres-ssf) [inline src/logic/operate.js]: - [x] As it said in the instructions, operate should accept 3 parameters: numberOne, numberTwo, and operator. The operate method is going to be called by the calculate method when the user inputs an operation to happen.
Screenshot from 2021-01-12 14-01-21

- [x] The modifier operator here is supposed to work differently from the `mod`'s Big.js. Mod here is supposed to turn the number into a percentage based number, e.g. `95 % == .95`
Screenshot from 2021-01-12 14-08-26

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const calculate = (dataObjt, buttonName) => {
+  let { total = '', next = '', operation = '' } = dataObjt;
+  switch (buttonName) {
+    case 'AC':
+      total = '';
+      next = '';
+      operation = '';
```
Comment (Torres-ssf) [inline src/logic/calculate.js:9]: - [x] Since operate is going to be used here depending on the user's input, we need to import the operate function here.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hi @elberthcorniell :hand: 
Awesome job on this milestone so far. But we need to make soma changes to it before moving on.

- [x] Check my comments below.

OPTIONAL

- [x] There are some unnecessary files here that we were supposed to delete on Milestone 1.

- The `logo.svg` file it's not being used in the project. We can safely delete it.
- The App.test.js file it's a boilerplate test file generated by create-react-app. It can also be safely delete.

Feel free to message me on Slack if you have any questions about it.
Happy coding :computer: 
**Sergio Torres**

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @elberthcorniell :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**